### PR TITLE
feat(config, logger): Add config and logging modules

### PR DESCRIPTION
**Pull request by chriswmackey**
I added this mostly so that we use a consistent location to download EPW files to. I have verified that this module will work well on both Windows and Unix-based systems.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,7 +0,0 @@
-FROM python:2.7
```
Comment (mostaphaRoudsari) [inline Dockerfile]: Why are we deleting the docker file?

Comment (chriswmackey) [reply Dockerfile]: Ah sorry. I just saw that none of the other repos had it and I wasn't sure why it was there. I will add it back. Do we need to add a Dockerfile to the other repos?

Comment (mostaphaRoudsari) [reply Dockerfile]: Eventually yes. We want to have a docker image for each of the libraries that one can pull down and use. We are adding them one by one.

Comment (chriswmackey) [reply Dockerfile]: Understood. Thank you for clarifying.

**Pull request merged**